src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -13,17 +13,26 @@
 #    limitations under the License.
 
 from full_activity_dump_helper import getFullActivityDumpSasTestHarness, getFullActivityDumpSasUut
+from functools import partial
 import json
 import os
 import sas
 import sas_testcase
 from sas_test_harness import SasTestHarnessServer, generateCbsdRecords, \
     generatePpaRecords
+from reference_models.pre_iap_filtering import pre_iap_filtering
 import time
 import test_harness_objects
 from util import winnforum_testcase, configurable_testcase, getCertificateFingerprint, writeConfig, \
   loadConfig, makePpaAndPalRecordsConsistent
+from full_activity_dump import FullActivityDump
+from reference_models.iap import iap 
+from reference_models.interference import aggregate_interference, interference
+import multiprocessing
```

Comment:
Replace this line and following with:
  from reference_models.common import mpool

All multiprocessing stuff are now centralized in that module and the iap and other modules use that mpool.Pool().
This allows to configure the test harness in one single place. See also last pending item on this in #580

Reply:
Replaced   the import